services/dynamic-sidecar/Dockerfile

```diff
@@ -22,6 +24,19 @@ RUN set -eux && \
   # verify that the binary works
   gosu nobody true
 
+# Install docker
+ENV DOCKER_VERSION="5:20.10.16~3-0~debian-buster"
```

Comment:
why do you need to install docker here? would it nos suffice using ``aiodocker``? This container has already ``/var/run/docker.sock`` mount